google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsAttemptCallable.java

```diff
@@ -230,14 +236,16 @@ private void handleAttemptError(Throwable rpcError) {
     Builder builder = lastRequest.toBuilder().clearEntries();
     List<Integer> newOriginalIndexes = Lists.newArrayList();
 
+    TimedAttemptSettings nextAttemptSettings =
```

Comment:
why do you need a local var for this? why not jsut assign to the instance var attemptSettings?